### DISCUSSION TITLE
wpf类库项目或者用户控件库项目使用handycontrol

### DISCUSSION THREAD

**Discussion by jinleil** [HandyOrg/HandyControl#1491] [Q&A]
wpf类库项目或者用户控件库项目怎么引用handycontrol呢？有没有官方的使用示例？

我的多个应用程序有一些相同的页面，想把这些相同的页面封装在用户控件库中，但是在用户控件库项目中没有App.xaml文件，
那么想handycontrol的控件，该怎么添加资源引用呢？

**Comment by liping94** [comment]
如果引用当前用户控件的父控件或者窗体中引用了handyControl的资源，那么在运行时会传递给当前用户控件  只是你在编辑界面时没效果；
另外你也可以在类库中加一个资源字典，在资源字典中添加handyControl的资源引用，然后在控件自身的资源中引用当前资源字典；后者比较蠢，但是有用
